lib/train/transports/ssh.rb

```diff
@@ -193,6 +193,34 @@ def verify_host_key_option
       current_net_ssh >= new_option_version ? :verify_host_key : :paranoid
     end
 
+    # Likewise, version <5 accepted false; 5+ requires :never or will
+    # issue a deprecation warning. This method allows a lot of common
+    # things through.
+    def verify_host_key_value(given)
```

Comment:
Should this have tests?

Reply:
I've added some, but stubbing the const used to detect the net-ssh version (which affects the mapping, here) proved elusive. So, the tests check whatever net-ssh version is currently being used, which is not ideal.